<launch>
    <arg name="rviz" default="false"/>
    <node pkg="rviz" name="rviz_limo" type="rviz" 
          args="-d $(find fast_limo)/config/rviz/limo.rviz" if="$(arg rviz)"/>

    <group ns="ona2">
        <node pkg="fast_limo" type="fast_limo_reloca_exec" 
              name="fast_limo_reloca" output="screen">
            <param name="mode" value="false" />
            <param name="map_path" value="$(find fast_limo)/maps/campus_nord.pcd" />
            <param name="distance_threshold" value="10.0" />
            <param name="inliers_threshold" value="10" />
            <param name="score" value="10" />
        </node>
    </group>
</launch>
